components/script/dom/htmlinputelement.rs

```diff
@@ -1228,3 +1230,8 @@ impl Activatable for HTMLInputElement {
         }
     }
 }
+
+fn filter_from_accept(_s: DOMString) -> Vec<FilterPattern> {
+    unimplemented!()
```

Comment:
@Manishearth I contacted [upstream](https://github.com/cybergeek94/mime_guess/issues/19), and the functionality necessary for this is implemented, so I am waiting for him to upload the new version actually 😅